paddle/fluid/eager/auto_code_generator/generator/eager_gen.py

```diff
@@ -72,7 +72,7 @@
     "subtract_double_grad",
     "add_triple_grad",
     "silu_double_grad",
-    "tanh_double_grad",
```

Comment:
Why delete `tanh_double_grad`?

Reply:
When decomposing tanh_triple_grad, the tanh_grad and tanh_double_grad can use fused kernels for better performance, thus remove tanh_double_grad from prim_white_list to prevent decomposing tanh_double_grad redundantly.